src/dmd/backend/dwarfdbginf.d

```diff
@@ -1604,9 +1604,12 @@ static if (1)
             if (!sd.SDlinnum_data.length)
                 continue;
 
+            static if (0)
+            {
```

Comment:
Why the `static if(0)` and not just delete it altogether?

Reply:
I just wanted to see if it passed the tests before deleting it.